server/src/utils/auth.ts

```diff
@@ -59,3 +60,6 @@ export async function getUserInfo(
     userInfo: response.value.data,
   };
 }
+
+export const verifyTwilioEmail = (email: string): boolean =>
+  email.indexOf('@twilio.com') > -1;
```

Comment:
If you change the env to EMAIL_VERIFICATION_REGEX, this can be simplified
```ts
export const verifyTwilioEmail = (email: string, regex: string): boolean =>
  regex && new RegExp(regex).test(email);
```

Reply:
This way, we also don't have to hardcode `@twilio.com`